.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "spdk"]
 	path = spdk
-	url = https://github.com/ceph/spdk.git
-	branch = ceph-nvmeof
+	url = https://github.com/spdk/spdk.git
```

Comment:
Are we 100% sure about this? I see benefits, as it'll save 1 step in the upgrade process (updating the ceph/spdk fork), but it removes our ability to cherry-pick commits on top of the latest spdk/spdk.

An alternative approach would be to leverage the `rpmbuild` (containerized) process, which is essentially designed [to apply patches](https://rpm-software-management.github.io/rpm/manual/autosetup.html) on top of tarballed source tree.

Reply:
@epuertat 🖖 Pushed v23.01.1 tag to ceph/spdk see [here](https://github.com/ceph/spdk/tree/v23.01.1). This way we're still tracking https://github.com/ceph/spdk.git repo. Not sure about ceph-nvmeof branch destiny. WDYT?

Reply:
We should continue using ceph-nvmeof branch -- I have just updated it.

Reply:
@baum Please redo this PR so that `.gitmodules` is left intact.

Reply:
@idryomov ✅ Thank you for your help.